.github/workflows/main.yml

```diff
@@ -326,6 +326,9 @@ jobs:
       - name: test in dev mode
         run: RISC0_DEV_MODE=1 cargo test --locked -F $FEATURE --workspace --exclude prover-example
         working-directory: examples
+      - name: test JWT validator separately
+        run: RISC0_DEV_MODE=1 cargo test --locked -F $FEATURE
+        working-directory: examples/jwt-validator
       - name: test prover example
```

Comment:
we'll need to explicitly exclude this from the line above if we're going to do it separately, but why do we need to do this?